lib/config/migration.ts

```diff
@@ -33,7 +33,7 @@ export function migrateConfig(
         optionTypes[option.name] = option.type;
       });
     }
-    const newConfig = MigrationsService.run(config);
+    const newConfig = MigrationsService.run(config).migratedConfig;
```

Comment:
why you don't use the `isMigrated property?

Reply:
It is defined below in this file as `const isMigrated = !dequal(config, migratedConfig);`. For now `isMigrated` introduced inside `MigrationsService` for safely moving unit tests to separate files.